js/viz.js

```diff
@@ -325,13 +351,11 @@ const viz = {
 
   dragEnd() {
     d3.event.subject.shadow = false;
-    this.endEvent();
```

Comment:
@jonathanKingston > on update the whole graph is still recalculated

Do you mean, that when new nodes come in, the existing graph shouldn't change? And the graph should change/recalculated only during page refresh?